### PR TITLE
binpacking recipe

### PR DESCRIPTION
**Pull request by pirovc**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by benmaier** [comment]
I agree to be listed

**Comment by pirovc** [comment]
@conda-forge/help-python could you please review. Recipe generated from a pip package skeleton, just added the pytest to the host requirements.

File: recipes/binpacking/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+test:
+  imports:
+    - binpacking
+  commands:
+    - binpacking --help
+  requires:
+    - pytest
+    - pytest-cov
+    - pytest-runner
```
Comment (beckermr) [inline recipes/binpacking/meta.yaml]: Unless you are using pytest to actually run the tests, I don't think you need the packages here.

File: recipes/binpacking/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  host:
+    - pip
+    - python
+    - numpy
+    - future
+    - pytest
+    - pytest-cov
+    - pytest-runner
+  run:
+    - pip
```
Comment (chrisburr) [inline recipes/binpacking/meta.yaml]: I guess `pip` isn't a runtime dependency?
```suggestion
```

File: recipes/binpacking/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  entry_points:
+    - binpacking = binpacking.binpacking_binary:main
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir"
+
+requirements:
+  host:
+    - pip
+    - python
+    - numpy
```
Comment (chrisburr) [inline recipes/binpacking/meta.yaml]: Do you need `numpy` and `future` in the `host` section?

File: recipes/binpacking/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    - future
+    - pytest
+    - pytest-cov
```
Comment (chrisburr) [inline recipes/binpacking/meta.yaml]: Glancing at the `setup.py` script, I think you don't need `pytest` and `pytest-cov`?

File: recipes/binpacking/meta.yaml (hunk 3 above)
Comment (pirovc) [reply recipes/binpacking/meta.yaml]: you're right, done

File: recipes/binpacking/meta.yaml (hunk 4 above)
Comment (pirovc) [reply recipes/binpacking/meta.yaml]: you're right, done

File: recipes/binpacking/meta.yaml (hunk 2 above)
Comment (pirovc) [reply recipes/binpacking/meta.yaml]: you're right, done

File: recipes/binpacking/meta.yaml (hunk 1 above)
Comment (pirovc) [reply recipes/binpacking/meta.yaml]: you're right, done

**Pull request merged**